upstream-community-operators/dynatrace-operator/0.2.1/manifests/dynatrace-operator.v0.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,771 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "dynatrace.com/v1alpha1",
+          "kind": "DynaKube",
+          "metadata": {
+            "name": "dynakube",
+            "namespace": "dynatrace"
+          },
+          "spec": {
+            "activeGate": {
+              "image": ""
+            },
+            "apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+            "classicFullStack": {
+              "enabled": true,
+              "tolerations": [
+                {
+                  "effect": "NoSchedule",
+                  "key": "node-role.kubernetes.io/master",
+                  "operator": "Exists"
+                }
+              ]
+            },
+            "kubernetesMonitoring": {
+              "enabled": true
+            },
+            "oneAgent": {
+              "image": ""
+            },
+            "routing": {
+              "enabled": true
+            }
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Monitoring,Logging & Tracing
+    containerImage: docker.io/dynatrace/dynatrace-operator
```

Comment:
Can you fix version here? docker.io/dynatrace/dynatrace-operator

Reply:
@DTMad ^^^

Reply:
Sure!
Done @mvalarh